lib/lightning/version_control/project_repo.ex

```diff
@@ -0,0 +1,43 @@
+defmodule Lightning.VersionControl.ProjectRepo do
+  @moduledoc """
+  Ecto model for project repo connections
+  """
+
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  alias Lightning.Projects.Project
+  alias Lightning.Accounts.User
+
+  @type t() :: %__MODULE__{
+          __meta__: Ecto.Schema.Metadata.t(),
+          id: Ecto.UUID.t() | nil,
+          github_installation_id: String.t() | nil,
+          target_id: String.t() | nil,
+          repo: String.t() | nil,
+          branch: String.t() | nil,
+          project: nil | Project.t() | Ecto.Association.NotLoaded,
+          user: nil | User.t() | Ecto.Association.NotLoaded
+        }
+
+  @primary_key {:id, :binary_id, autogenerate: true}
+  @foreign_key_type :binary_id
+  schema "project_repos" do
+    field :github_installation_id, :string
+    field :target_id, :string
```

Comment:
What is `target_id` for?

Reply:
I think this is leftover from a previous attempt will remove